Paralayout/UIView+Alignment.swift

```diff
@@ -246,6 +270,20 @@ extension UIView {
         )
     }
 
+    /// Move the view to align it within its superview.
+    ///
+    /// - precondition: The receiver must have a superview.
+    ///
+    /// - parameter position: The position in both the receiving view and its `superview` to use for alignment.
+    /// - parameter offset: An additional offset to apply to the receiver.
+    public func alignToSuperview(_ position: Position, offset: UIOffset) {
```

Comment:
Does it make sense to provide a default offset as well? Then, we can just call `alignWithSuperview(.center)`

Reply:
`alignToSuperview(.center)` will still work, as it will call the `alignToSuperview(_:horizontalOffset:verticalOffset:)` variant.

Reply:
Aaah, sorry, I got lost 🤦‍♀️

Reply:
No worries! All of the similarly named methods get confusing.